hr_timesheet_task_domain/i18n/hr_timesheet_task_domain.pot

```diff
@@ -0,0 +1,20 @@
+# Translation of Odoo Server.
```

Comment:
can be removed

Reply:
Why? POT is re-generated any way on merge, although empty.